test/Plot/Functions/Plot1.purs

```diff
@@ -0,0 +1,120 @@
+module Test.Plot.Functions.Plot1
+  ( plot1Tests
+  ) where
+
+import Prelude
+import Plot.Functions (plot1)
+import Test.Unit (TestSuite, suite, test)
+import Test.Unit.Assert (equal)
+
+plot1Tests :: TestSuite
+plot1Tests =
```

Comment:
For this type of testing I hope we can use https://pursuit.purescript.org/packages/purescript-quickcheck/4.2.0 or similar.  Nevertheless, we will probably not test a specific function once we have support for user to enter arbitrary functions.  No need to change anything now yet, but would be good to start looking into QuickCheck.  I hope to soon add QuickCheck tests for Approx.